lib/runtime/ppx_mysql_runtime.ml

```diff
@@ -200,7 +200,6 @@ let int64_of_string_exn = wrap_deserializer Int64.of_string
 let bool_of_string_exn = function
   | "0" ->
       false
-  | "1" ->
-      true
-  | str ->
-      raise (Deserialization_error ("Ppx_mysql_runtime.bool_of_string_exn", str))
+  | str -> (
+    try int_of_string str <> 0 with Failure _ ->
+      raise (Deserialization_error ("Ppx_mysql_runtime.bool_of_string_exn", str)) )
```

Comment:
Personally I find

```ocaml
| str ->
  match int_of_string str <> 0 with
  | v -> v
  | exception Failure _ -> raise (Deserialization_error ("Ppx_mysql_runtime.bool_of_string_exn", str))
```

A lot easier to read and understand what is happening. Also avoids parens.